packages/desktop/components/SidebarTab.svelte

```diff
@@ -15,6 +15,12 @@
         addToVisitedTabs(tab?.route)
     }
 
+    $: {
```

Comment:
I think it would be better to add this in `implicitAccountCreationView` in the `handleMultipleAccounts` function, instead of a reactive conditional in this component. wdty?